### PR TITLE
Use a CSV upload to resync

### PR DESCRIPTION
**Pull request by zachlefevre**

### CODE REVIEW

File: store-redshift/src/main/scala/com/socrata/store/blob/DataDeleter.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import jakarta.enterprise.context.ApplicationScoped
+
+trait DataDeleter {
+  def delete(
+      key: String
+  ): Unit
+}
+
+@ApplicationScoped
+case class DataDeleterImpl(
```
Comment (PaulMTT) [inline store-redshift/src/main/scala/com/socrata/store/blob/DataDeleter.scala:13]: Not a deal breaker, but any reason to not have data operations be within the same interface? DataStorage+DataDeleter?

File: store-redshift/src/main/scala/com/socrata/store/db/datasets/TablePopulator.scala [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import com.socrata.datacoordinator.id._
+import io.agroal.api.AgroalDataSource
+import io.quarkus.agroal.DataSource
+import com.socrata.datacoordinator.secondary._
+import com.socrata.soql.types._
+import com.socrata.soql.analyzer2._
+
+@jakarta.enterprise.context.ApplicationScoped
+case class TablePopulator(
```
Comment (PaulMTT) [inline store-redshift/src/main/scala/com/socrata/store/db/datasets/TablePopulator.scala:11]: Same here with the split class operators.

**Review by PaulMTT — APPROVED** [review]

File: store-redshift/src/main/scala/com/socrata/store/blob/DataDeleter.scala (hunk 1 above)
Comment (zachlefevre) [reply store-redshift/src/main/scala/com/socrata/store/blob/DataDeleter.scala:13]: They have different responsibilities, IMO.

File: store-redshift/src/main/scala/com/socrata/store/db/datasets/TablePopulator.scala (hunk 2 above)
Comment (zachlefevre) [reply store-redshift/src/main/scala/com/socrata/store/db/datasets/TablePopulator.scala:11]: Same here. They have different dependencies and different responsibilities. It seems unreasonable to me to require passing in dependencies for functionality that makes no use of those dependencies.

File: store-redshift/src/main/scala/com/socrata/store/blob/DataDeleter.scala (hunk 1 above)
Comment (PaulMTT) [reply store-redshift/src/main/scala/com/socrata/store/blob/DataDeleter.scala:13]: To me it makes sense to bundle all the crud methods together, but like I said no big deal this still works great.

File: store-redshift/src/main/scala/com/socrata/store/db/datasets/TablePopulator.scala (hunk 2 above)
Comment (PaulMTT) [reply store-redshift/src/main/scala/com/socrata/store/db/datasets/TablePopulator.scala:11]: Yeah actually this one I see as more appropriate than the s3 stuff.

**Pull request merged**